src/window-system/x11-cairo-surface.cpp

```diff
@@ -57,6 +57,14 @@ X11CairoSurface::X11CairoSurface(Display *display) : display(display) {
       CWColormap | CWBorderPixel | CWBackPixel | CWOverrideRedirect, &attr);
   XMapWindow(display, this->window);
 
+  // Add WM_NAME and WM_CLASS properties to the window
+  std::string name("touchégg");
+  XStoreName(display, this->window, &name[0]);
```

Comment:
You can use `c_str` to get a const char *:

```suggestion
  XStoreName(display, this->window, name.c_str());
```